editor/src/core/model/element-metadata-utils.ts

```diff
@@ -413,18 +412,50 @@ export const MetadataUtils = {
       return null
     }
   },
-  getFlexDirection: function (instance: ElementInstanceMetadata | null): string {
+  getFlexDirection: function (instance: ElementInstanceMetadata | null): FlexDirection {
     return instance?.specialSizeMeasurements?.flexDirection ?? 'row'
   },
   getSimpleFlexDirection: function (instance: ElementInstanceMetadata | null): {
-    direction: SimpleFlexDirection | null
-    forwardsOrBackwards: FlexForwardsOrBackwards | null
+    direction: Direction
+    forwardOrReverse: ForwardOrReverse
   } {
-    // TODO move the actual helper functions here
-    const direction = MetadataUtils.getFlexDirection(instance)
+    return MetadataUtils.flexDirectionToSimpleFlexDirection(
+      MetadataUtils.getFlexDirection(instance),
+    )
+  },
+  flexDirectionToSimpleFlexDirection: function (flexDirection: FlexDirection): {
+    direction: Direction
+    forwardOrReverse: ForwardOrReverse
+  } {
+    const direction: Direction = (() => {
```

Comment:
Weren't these two blocks in self contained functions before?

Reply:
they were! my thinking was that it's better for everyone if I consolidate the 3-4 competing ways of achieving the same task, so I made these functions "really private" by inlining them here